requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests
 bs4
 lxml
-cli
+click
+pandas
```

Comment:
Cant you not use pandas?

Reply:
your python program already uses pandas . I just added to requirements.txt , take a look at your program files